src/test/java/com/woowacamp/soolsool/integration/repository/LiquorStatusTypeRepositoryTest.java

```diff
@@ -20,7 +21,9 @@ class LiquorStatusTypeRepositoryTest {
     @DisplayName("LiquorStatus의 name를 가지고 LiquorStatus을 조회한다.")
     void findByLiquorRegionType_type() {
         // given
-        LiquorStatus 판매중 = liquorStatusRepository.findByType(ON_SALE);
+        LiquorStatus 판매중 = liquorStatusRepository
+            .findByType(ON_SALE).orElseThrow();
```

Comment:
첫 줄엔 findByType을 두고 orElseThrow는 두 번째 줄에 있었으면 좋겠어요!!!!
```suggestion
        LiquorStatus 판매중 = liquorStatusRepository.findByType(ON_SALE)
            .orElseThrow();
```